File: tests/fixtures/php/sample.php
<?php

namespace App\Calc;

use App\Contracts\Adder;
use App\Util\Logger;

/**
 * Calculator doc block.
 */
class Calculator implements Adder
{
    // running total
    private int $total = 0;

    public function add(int $lhs, int $rhs): int
    {
        return $lhs + $rhs;
    }

    public function describe(): string
    {
        $banner = <<<TXT
        Calculator
        ==========
        TXT;
        return $banner;
    }
}

function helper(string $name): string
{
    # greet the caller
    $multiply = function (int $a, int $b): int {
        return $a * $b;
    };

    return "Hello, {$name}, this is intentionally long enough to truncate past the limit";
}

class CalculatorTest extends TestCase
{
    public function testAdds(): void
    {
        $calc = new Calculator();
        $this->assertSame(3, $calc->add(1, 2));
    }
}

File: tests/fixtures/go/sample.go
package calc

import (
	"fmt"
	"strings"
)

// Running total seed.
const Seed = 0

// Calculator sums integers.
type Calculator struct {
	total int
}

func (c *Calculator) Add(lhs, rhs int) int {
	return lhs + rhs
}

func (c *Calculator) Describe() string {
	banner := "Calculator"
	return banner
}

func Helper(name string) string {
	return fmt.Sprintf("Hello, %s, this banner is intentionally long enough to truncate past the configured default string limit of two hundred and fifty six bytes so the truncation marker is emitted in the golden snapshot output here for the go fixture.", name)
}

func TestAdd(t *testing.T) {
	c := &Calculator{}
	if c.Add(1, 2) != 3 {
		t.Fatal("bad")
	}
}
